src-docs/src/views/guidelines/accessibility.js

```diff
@@ -510,6 +510,56 @@ export default {
         </>
       ),
     },
+    {
+      title: 'Disabled elements',
+      wrapText: false,
+      text: (
+        <>
+          <EuiText grow={false}>
+            <p>
+              If you disable elements by adding the HTML{' '}
+              <EuiCode language="html" size="s">
+                disabled
+              </EuiCode>{' '}
+              attribute or passing the{' '}
+              <EuiCode language="js">isDisabled</EuiCode> prop to the{' '}
+              <EuiLink href="http://localhost:8030/#/layout/accordion#disabled-state">
```

Comment:
This needs to be a Link element imported from `react-router-dom` to work correctly
```suggestion
              <Link to="/layout/accordion#disabled-state">
```